src/server/handlers.rs

```diff
@@ -0,0 +1,24 @@
+use actix_web::{HttpRequest, HttpResponse, Responder};
+
+pub async fn get_handler(_req: HttpRequest) -> impl Responder {
```

Comment:
Don't forget to wire this new handler with a route to it in `src/server/mod.rs`.

Reply:
Sorry, I forgot that. I have made the changes.